qiskit_experiments/library/characterization/fine_amplitude.py

```diff
@@ -0,0 +1,303 @@
+
+    # section: overview
+
+        :class:`FineXAmplitude` is a subclass of :class:`FineAmplitude` and is used to set
+        the appropriate values for the default options.
+    """
+
+    def __init__(self, qubit: int):
+        """Initialize the experiment."""
+        super().__init__(qubit, XGate())
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        r"""Default values for the fine amplitude experiment.
+
+        Experiment Options:
+            gate (Gate): FineXAmplitude calibrates an XGate.
+            add_sx (bool): This option is True by default when calibrating gates with a target
+                angle per gate of :math:`\pi` as this increases the sensitivity of the
+                experiment.
+            add_xp_circuit (bool): This option is True by default when calibrating gates with
+                a target angle per gate of :math:`\pi`.
+        """
+        options = super()._default_experiment_options()
+        options.gate = XGate()
+        options.add_sx = True
+        options.add_xp_circuit = True
+
+        return options
+
+    @classmethod
+    def _default_analysis_options(cls) -> Options:
+        """Default analysis options."""
+        options = super()._default_analysis_options()
+        options.angle_per_gate = np.pi
+        options.phase_offset = np.pi / 2
+
+        return options
+
+
+class FineSXAmplitude(FineAmplitude):
+    r"""A fine amplitude experiment with all the options set for the :math:`\pi/2`-rotation.
+
+    # section: overview
+
+        :class:`FineSXAmplitude` is a subclass of :class:`FineAmplitude` and is used to set
+        the appropriate values for the default options.
+    """
+
+    def __init__(self, qubit: int):
+        """Initialize the experiment."""
+        super().__init__(qubit, SXGate())
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        r"""Default values for the fine amplitude experiment.
+
+        Experiment Options:
+            gate (Gate): FineSXAmplitude calibrates an SXGate.
+            add_sx (bool): This option is False by default when calibrating gates with a target
+                angle per gate of :math:`\pi/2` as this increases the sensitivity of the
+                experiment.
+            add_xp_circuit (bool): This option is False by default when calibrating gates with
+                a target angle per gate of :math:`\pi/2`.
+            repetitions (List[int]): By default the repetitions take on odd numbers for
+                :math:`\pi/2` target angles as this ideally prepares states on the equator of
+                the Bloch sphere. Note that the repetitions include two repetitions which
+                plays the same role as including a circuit with an X gate.
+        """
+        options = super()._default_experiment_options()
+        options.gate = SXGate()
+        options.add_sx = False
```

Comment:
Do we want to support the user changing ``add_sx` and `add_xp_circuit`? As long as the analysis options are changed in sync, it works okay but the point of this subclass is to freeze those options. @nkanazawa1989 I don't know if this use case was considered in the autodoc code. Would it make sense to support a subclass hiding some options of a parent class when the subclass is fixing them to specific values?